recipes/flowermd/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "flowermd" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/cmelab/flowerMD/archive/{{ version }}.tar.gz
+  sha256: e4b2857c58d9f8a9e4863b789224bdfd9325a703dc5c3482f11b16adc1a621dd 
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  noarch: python
+  skip: true  # [win or py<36 or python_impl == "pypy"]
```

Comment:
```suggestion
```

Reply:
@dbast I just wanted to follow up on this. I think `flower` should technically be no arch, its a pure python package, but the windows build fails because of other dependencies. Reading the [documentation about this](https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-packages-with-os-specific-dependencies) it sounds like the correct thing to do is to set `noarch: python` in the `meta.yml`: 

> noarch is a statement about the package’s source code and not its install environment. A package is still considered noarch even if one of its dependencies is not available on a given platform.


But, this means the windows build in the CI will fail. Do we just merge it anyway, and future CI builds will only use linux?

Reply:
This one can be noarch even if Windows fails due to a missing dependency. We save CI resources and the Windows side won't get installed due to the missing dep. However, when the dependency is available, no change is needed here to make it installable.